autoload/phpcd.vim

```diff
@@ -903,8 +903,13 @@ function! phpcd#GetClassName(start_line, context, current_namespace, imports) "
 			if line =~? '^\s*' . object . '\s*=.*);\?$' " {{{
 				let classname = phpcd#GetCallChainReturnTypeAt(a:start_line - i)
 				let classname_parts = split(classname, '\\\+')
```

Comment:
Only when `classname` is empty, could we got the `classname_parts[-1] out of bound` error. However, I could not see there is any case to let `classname` to be empty. Please offer more detail to recur this problem. Thanks.